tdrs-backend/tdpservice/email/templates/request-approved.html

```diff
@@ -4,7 +4,11 @@
 <p style="color: #000000;">
     <p>Dear {{ first_name }},</p>
 
-    <p>Congratulations! Your request for the TANF Data Portal (TDP) is <b>approved</b> and access as a {{ group_permission }} from {{ stt_name }} <b>has been granted</b>.</p>
+    {% if regional %}
+        <p>Congratulations! Your request for the TANF Data Portal (TDP) is <b>approved</b> and access as a {{ group_permission }} for Region(s) {{ regions }} <b>has been granted</b>.</p>
```

Comment:
regions as an array comes across a little funny
<img width="373" alt="image" src="https://github.com/user-attachments/assets/46318c29-fd3b-4fdc-944c-bc8cc83b9284" />
could we use the `join` template tag filter? https://docs.djangoproject.com/en/5.1/ref/templates/builtins/#std-templatefilter-join